extension/src/plots/model/index.ts

```diff
@@ -198,12 +193,12 @@ export class PlotsModel extends ModelWithPersistence {
   public getSelectedRevisionDetails() {
     const selectedRevisions: Revision[] = []
     for (const experiment of this.experiments.getSelectedRevisions()) {
-      const { commit, displayName, label, displayColor, logicalGroupName, id } =
+      const { commit, description, label, displayColor, logicalGroupName, id } =
         experiment
       const revision: Revision = {
         displayColor,
-        errors: this.errors.getRevisionErrors(label),
-        fetched: this.fetchedRevs.has(label),
+        errors: this.errors.getRevisionErrors(id),
```

Comment:
[F] The reason that we have to switch from revision to id for identifying an experiment is that the revision (aka label) on an experiment can be duplicated for experiments that are running in the workspace (the label is also workspace).